### PR TITLE
[FEATURE] Demonstrate all possible site setting types

### PR DESCRIPTION
**Pull request by linawolf**
Related: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1026
Releases: main

### CODE REVIEW

File: Configuration/Sets/Examples/settings.definitions.yaml [hunk 1]
```diff
@@ -1,31 +1,111 @@
...
+    label: 'Examples'
+    parent: Example
+  BlogExample.types:
+    label: 'Available types'
+    parent: Example
+
 settings:
   example.output.view.templateRootPath:
     default: 'EXT:example/Resources/Private/Templates/'
+    category: Example.examples
```
[the remark this answers is not in the document]
Comment (linawolf) [reply Configuration/Sets/Examples/settings.definitions.yaml:14]: Where are they not rendered? The TYPO3 Backend?

[the remark this answers is not in the document]
Comment (froemken) [reply Configuration/Sets/Examples/settings.definitions.yaml:14]: There is not category column in menu and no category output for individual property in docs:

Bildschirmfoto 2024-09-26 um 23 08 00

[the remark this answers is not in the document]
Comment (linawolf) [reply Configuration/Sets/Examples/settings.definitions.yaml:14]: True, could you open a ticket in the render-guides please?

[the remark this answers is not in the document]
Comment (froemken) [reply Configuration/Sets/Examples/settings.definitions.yaml:14]: Done: https://github.com/TYPO3-Documentation/render-guides/issues/772

[the remark this answers is not in the document]
Comment (linawolf) [reply Configuration/Sets/Examples/settings.definitions.yaml:14]: @froemken please render again locally, the categories should be displayed now

**Pull request merged**